application/config/config.go

```diff
@@ -866,6 +884,28 @@ func (c *Config) IsFedramp() bool {
 	host := strings.ToLower(u.Hostname())
 
 	// fedramp instance should have the format https://*.snykgov.io
-	snykgovDomain := strings.HasSuffix(host, ".snykgov.io")
-	return snykgovDomain
+	return isFedrampHost(host)
+}
+
+func (c *Config) GetSnykCodeApi() (string, error) {
```

Comment:
We have `SnykCodeApi` getter already at line :338.

Reply:
We do, but that is used by the sast_local_engine, and there is no error handling there, so I didn't want to introduce one. I thought the local_engine can use the original SnykCodeApi getter, and the backend_service will use the new one, that can also return an error.